qiita_db/metadata_template.py

```diff
@@ -1965,7 +1993,29 @@ def load_template_to_dataframe(fn):
                            keep_default_na=False, na_values=[''],
                            parse_dates=True, index_col=False, comment='\t',
                            mangle_dupe_cols=False, converters={
-                               'sample_name': lambda x: str(x).strip()})
```

Comment:
I wanted to do something like that but I ended up opting for this solution. The reason being that pandas will try to infer data types (which is useful) unless you want to keep the string as is. If you see the types for latitude/longitude are not enforced in this dictionary, instead I let pandas infer those types and if those are not inferred as floats, we raise an error.

Since this is a two-step process I don't think there's a good way to automate this.